db/inmemory/client.go

```diff
@@ -0,0 +1,23 @@
+package inmemory
+
+import (
+	"github.com/go-swagger/go-swagger/examples/accountmgr/db"
+	"github.com/go-swagger/go-swagger/examples/accountmgr/domain"
+)
+
+func init() {
+	db.RegisterDataStore("inmemory", NewClient)
+}
+
+func NewClient() (db.DataStore, error) {
+	return &client{ds: make(map[string]*domain.Account)}, nil
+}
+
+type client struct {
+	ds map[string]*domain.Account
+}
+
+func (c *client) AddAccount(user *domain.Account) (string, error) {
+	c.ds[user.UserID] = user
```

Comment:
it should store based on account.Id

Reply:
Added id and name for account and done the above on the basis of accountid